Sources/Web5/Crypto/JOSE/JWT.swift

```diff
@@ -26,33 +26,38 @@ public struct JWT {
         /// or after which the JWT must not be accepted for processing.
         ///
         /// [Spec](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4)
-        @ISO8601Date private(set) var expiration: Date?
```

Comment:
are date fields in jwt claims supposed to be a numeric value? is there any value in saving it as a date?

Reply:
https://github.com/TBD54566975/web5-swift/issues/46